services/middleware/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "exonum-middleware-service"
+version = "0.13.0-rc.2"
+edition = "2018"
+authors = ["The Exonum Team <exonum@bitfury.com>"]
+homepage = "https://exonum.com/"
+repository = "https://github.com/exonum/exonum"
+documentation = "https://docs.rs/exonum-middleware-service"
+readme = "README.md"
+license = "Apache-2.0"
+keywords = ["exonum", "batch", "versioning"]
+categories = ["cryptography"]
+description = "Composability utilities for Exonum, packaged as a service"
+
+[badges]
+travis-ci = { repository = "exonum/exonum" }
+
+[dependencies]
+exonum = { version = "0.13.0-rc.2", path = "../../exonum" }
+exonum-crypto = { version = "0.13.0-rc.2", path = "../../components/crypto" }
```

Comment:
Since `exonum` provides public reexports for `crypto` and `merkledb`, can we avoid having them as dependencies?

Reply:
This dependencies are needed in the `exonum-derive` macro.

Reply:
Can you please create an issue for fixing it? IMO it's too indirect and hacky.

Reply:
Huh, it turns out that removing these dependencies makes the build script fail, of all things:

```text
thread 'main' panicked at 'Failed to get exonum crypto protobuf path: NotPresent'
...
  11: exonum_build::get_exonum_protobuf_crypto_files_path
  12: exonum_build::ProtoSources::path
```

@aleksuss Is this a known problem? Should I add a task for this issue?

Reply:
Interesting. Create issue please.

Reply:
@skletsun looks like familiar problem.

Reply:
Yes, I faced probably the same issue in EJB - https://jira.bf.local/browse/ECR-4036